go/libkb/all_provisioned_usernames.go

```diff
@@ -0,0 +1,59 @@
+package libkb
+
+func getUsernameIfProvisioned(m MetaContext, uc UserConfig) (ret NormalizedUsername, err error) {
+	m.CDebugf("getUsernameIfProvisioned(%+v)", uc)
+	did := uc.GetDeviceID()
+	if did.IsNil() {
+		m.CDebugf("- no valid username since nil deviceID")
+		return ret, nil
+	}
+	err = checkDeviceValidForUID(m.Ctx(), m.G().GetUPAKLoader(), uc.GetUID(), did)
+	switch err.(type) {
+	case nil:
+		m.CDebugf("- checks out")
+		return uc.GetUsername(), nil
+	case DeviceNotFoundError:
+		m.CDebugf("- user was likely reset (%s)", err)
+		return ret, err
```

Comment:
isn't this propagating all the way up?  and you just want to skip over this one in the loop?

Reply:
yeah, my bad. refresh, thank you.